addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -214,8 +216,15 @@ export class WebsocketWorker {
      *     - Number: user is logged whether on the frontend/backend.
      *     - false: user is not logged.
      *     - undefined: not available (e.g. livechat support page)
+     * @param {Number} param0.startTs Timestamp of start of bus service sender.
      */
-    _initializeConnection(client, { debug, lastNotificationId, uid, websocketURL }) {
+    _initializeConnection(client, { debug, lastNotificationId, uid, websocketURL, startTs }) {
+        if (this.newestStartTs && this.newestStartTs > startTs) {
+            this.isDebug = Object.values(this.debugModeByClient).some(debugValue => debugValue !== '');
```

Comment:
This line is missing :)
```js
this.debugModeByClient[client] = debug;
```

Reply:
done

Reply:
`WORKER_VERSION` should be incremented in order to force cache udpate.